core/src/test/java/io/questdb/test/griffin/JsonPlanSinkTest.java

```diff
@@ -78,14 +96,38 @@ public void toPlan(PlanSink sink) {
             sink.val(1L, 2L, 3L, 4L);
             sink.attr("plan");
             sink.val((Plannable) null);
-            sink.attr("double");
-            sink.val(0.0f);
-            sink.attr("float");
-            sink.val(1.0d);
             sink.attr("uuid");
             sink.valUuid(1L, 2L);
             sink.attr("uuid_null");
             sink.valUuid(Numbers.LONG_NaN, Numbers.LONG_NaN);
         }
     }
+
+    static class NumericalTestFactory implements RecordCursorFactory {
```

Comment:
This test might just be redundant because all the functionality is also tested somewhere in `core/src/test/java/io/questdb/test/griffin/ExplainPlanTest.java`. Should it stay?

Reply:
hey @biosfood thank you for the contribution, this can stay!